<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shopListDetail</class>
 <widget class="QWidget" name="shopListDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>741</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(22, 32, 45);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="discount_fra">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(76, 107, 34);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="discoutPrecent_lb">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>阿里巴巴普惠体 M</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-30%</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="proName_lb">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 M</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>商品名字</string>
    </property>
   </widget>
   <widget class="QFrame" name="proIcon">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Number_lb">
    <property name="geometry">
     <rect>
      <x>584</x>
      <y>80</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 M</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>商品数量</string>
    </property>
   </widget>
   <widget class="QLabel" name="discountPriceold_lb">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 M</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>商品价钱前</string>
    </property>
   </widget>
   <widget class="QLabel" name="discountPricenew_lb">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 M</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>商品价钱后</string>
    </property>
   </widget>
   <widget class="QLabel" name="lastNumber_lb">
    <property name="geometry">
     <rect>
      <x>584</x>
      <y>25</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 M</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>商品数量</string>
    </property>
   </widget>
   <widget class="QLabel" name="buytext_lb">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 M</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>购买量：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lastText_lb">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>25</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>阿里巴巴普惠体 M</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>剩余量：</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
